packages/dd-trace/src/span_sampler.js

```diff
@@ -73,8 +73,9 @@ class SpanSampler {
 
     const { started } = spanContext._trace
     for (const span of started) {
-      const tags = span.context()._tags || {}
-      const name = span._name
```

Comment:
Not a blocker as this is out of the scope of this PR, but we might want to remove this property completely as I don't think it's used anywhere? And if it is it could probably use a better name.

Reply:
What do you think of the additional commit I just made? That should solve the problem.

Reply:
I've replaced that commit with one explaining why `_name` exists and pointing to `context()._name` as the more likely thing one would be looking for.

I also set `_duration` to `undefined` in the constructor to avoid multiple class shapes.